src/store/reducers/user.js

```diff
@@ -1,10 +1,19 @@
 import { Map } from 'immutable';
 import { USER_SET_USER } from '../constants';
 
-const userReducer = (state = Map(), { type, payload = {} }) => {
+const initialState = Map({
+  name: 'Anonymous Name',
+  email: 'anonymous@gmail.com',
+  photo: '',
```

Comment:
I personally don't like these kinds of values. Also, that email actually exists. I'd recommend leaving the fields empty instead.